sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/artifactory.yaml

```diff
@@ -0,0 +1,82 @@
+provider:
+  metadata:
+    name: JFrog
+    category: Code hosting
+    message: Make sure this Artifactory token gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: artifactory-api-key
+      rspecKey: S6752
+      metadata:
+        name: Artifactory API key
+      detection:
+        pre:
+          include:
+            content:
+              - AKCp
+        matching:
+          pattern: "\\b(AKCp\\d[A-Za-z0-9_-]{68})\\b"
```

Comment:
Can be simplified with `[\\w-]`

Reply:
I'd rather not do it here (and after) I would definitely simplify `[A-Za-z_]` with `\w` but, in this case, I think that having the `_` inside `\w` and `-` outside of it, is not very clear. Moreover, I feel it does not really match the meaning of `\w`.